[DOCS] Removing Model Optimizer mentions from docs for master (#25505)
openvinotoolkit/openvino: docs/articles_en/documentation/openvino-extensibility/frontend-extensions.rst

The mapping doesn’t involve any attributes, as operation Identity doesn’t have them.

Extension objects, like just constructed ``extension`` can be used to add to the
OpenVINO runtime just before loading a model that contains custom operations:

.. doxygensnippet:: docs/articles_en/assets/snippets/ov_extensions.cpp
   :language: cpp
   :fragment: [frontend_extension_read_model]

> Or extensions can be constructed in a separately compiled shared library. Such library should be
> construced in case of loading models with custom operations in a Python application or tools like ``benchmark_app``
> For details on how to build and load such library, check the following :ref:`guide <create_a_library_with_extensions>`.

Reviewer: ```suggestion
However, extensions can also be constructed in a separately compiled shared library, that is suitable for loading models with custom operations in a Python application or tools like ``benchmark_app``.
For details on how to build and load such library, check the following :ref:`guide <create_a_library_with_extensions>`.
```
